Tabloid/client/src/modules/tagManager.js

```diff
@@ -0,0 +1,22 @@
+const _apiUrl = "/api/tag";
+
+export const getAllTags = () => {
+    return fetch(`${_apiUrl}/Index`)
```

Comment:
this method should be using the `getToken` function from the authManager to pass a token back to the server side.